lib/charms/loki_k8s/v0/loki_push_api.py

```diff
@@ -1996,7 +2002,11 @@ def _push_binary_to_workload(self, binary_path: str, workload_binary_path: str)
         """
         with open(binary_path, "rb") as f:
             self._container.push(
-                workload_binary_path, f, permissions=0o755, encoding=None, make_dirs=True
+                workload_binary_path,
+                f,
+                permissions=0o755,
+                encoding=None,  # pyright: ignore
```

Comment:
Do we need to raise an issue against ops?

Reply:
Might be a good idea!